src/main/java/com/auto/gen/junit/autoj/scanner/ClassScannerServiceImpl.java

```diff
@@ -80,7 +71,7 @@ private void processFiles(File directory, List<String> classes) throws FileNotFo
     private boolean isDtoOrEntityClass(CompilationUnit cu) {
         return cu.findAll(ClassOrInterfaceDeclaration.class)
                 .stream()
-                .anyMatch(c -> c.isAnnotationPresent("Entity") || c.isAnnotationPresent("Getter"));
+                .anyMatch(c -> c.isAnnotationPresent("Entity") || c.isAnnotationPresent("Getter")|| c.isAnnotationPresent("RestController")|| c.isAnnotationPresent("Controller")||c.isAnnotationPresent("Service"));
```

Comment:
segregate this to a different method as this DTO&Entity check is used in writer class